src/data/store.js

```diff
@@ -0,0 +1,11 @@
+import { applyMiddleware, createStore } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+
+import rootReducer from './reducers';
+
+const store = createStore(
+  rootReducer,
+  applyMiddleware(thunkMiddleware),
```

Comment:
Thunks aren't supported out of the box -- we have to enable them via middleware.